src/components/GradeLevelFilter.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button, Popover } from '@material-ui/core';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+
+const useStyles = makeStyles((theme) => ({
+  buttonStyle: {
+    margin: theme.spacing(1),
+    height: theme.spacing(4),
+  },
+  popOverWindow: {
+    minWidth: '200px',
+  },
+}));
+
+const gradeList = {
+  12: {
+    value: 'Highschool Senior',
+    isChecked: false,
+  },
+  13: {
+    value: 'College Freshman',
+    isChecked: false,
+  },
+  14: {
+    value: 'College Sophomore',
+    isChecked: false,
+  },
+  15: {
+    value: 'College Junior',
+    isChecked: false,
+  },
+  16: {
+    value: 'College Senior',
+    isChecked: false,
+  },
+};
+
+export default function GradeLevelFilter() {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const handleClick = (event) => setAnchorEl(event.currentTarget);
+
+  function updateGradeSelection(value) {
+    gradeList[value].isChecked = !gradeList[value].isChecked;
+    setAnchorEl(false);
```

Comment:
This closes the popover, which might be annoying if people want to select more than one check box. Remove?

Reply:
Sure, I did this for two reasons.
1. It took care of the rendering of check box not occur until the popover closes &
2. Technically a user would always need to only check 1 box for their specific grade, then if they are for example parents searching for multiple students then they would just go back and select the other grade
For now ill remove and we can figure out if it should be closed after a selection or kept open until user decided to close popover

Reply:
> 1. It took care of the rendering of check box not occur until the popover closes &

This is why you use add [state](https://reactjs.org/docs/state-and-lifecycle.html) to the component. When the state changes the component re-renders.

2. Technically a user would always need to only check 1 box for their specific grade, then if they are for example parents searching for multiple students then they would just go back and select the other grade

That's reasonable. Though in that case I think we can just provide an experience where if they click anywhere outside the Popover it closes (which makes things easier if they apply multiple filters)